firmware/quintuna/RSM/boot/hw_sysviewDescConfig.c

```diff
@@ -0,0 +1,7 @@
+#include "SEGGER_SYSVIEW.h"
+
+void hw_sysviewConfig_sendSystemDesc(void)
+{
+    SEGGER_SYSVIEW_SendSysDesc("N=RSM_Boot,D=Cortex-M4,O=FreeRTOS");
+    SEGGER_SYSVIEW_SendSysDesc("I#15=SysTick");
+}
```

Comment:
Would be nice to make a standard file to F4 vs. H7 variants since the interrupt names should be the same. Not relevant here but probably a good idea

Reply:
oh that is true. I was going to do that but then you would have to configure all the interrupts in the table which is hard

Reply:
actually you cannot do this because then all interrupts (whether or not it is caused) will be listed in systemview which is annoying